include/aikido/constraint/NonColliding.hpp

```diff
@@ -50,10 +50,21 @@ class NonColliding : public Testable
       std::shared_ptr<dart::collision::CollisionGroup> _group1,
       std::shared_ptr<dart::collision::CollisionGroup> _group2);
 
+  /// Remove collision check between group1 and group2.
+  /// \param group1 First collision group.
```

Comment:
Nit: The parameter names in this docstring are not consistent with the actual parameter names. I believe the convention is not to use a leading underscore for parameter names, but it seems this file doesn't comply. Let's fix this later as wholesale.